SYCL/Plugin/interop-level-zero-interop-task-mem.cpp

```diff
@@ -0,0 +1,65 @@
+// REQUIRES: level_zero, level_zero_dev_kit
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %level_zero_options %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+
+// Test for Level Zero interop_task.
+
+#include <CL/sycl.hpp>
+// clang-format off
+#include <level_zero/ze_api.h>
+#include <CL/sycl/backend/level_zero.hpp>
+// clang-format on
```

Comment:
Could you clarify why clang-format comments are needed here, please?

Reply:
clang-format puts <level_zero/ze_api.h> after <CL/sycl/backend/level_zero.hpp> and compilation fails because `backend/level_zero.hpp` uses Level-zero types and expects `ze_api.h` included before

Reply:
I see. Alternative way to handle this is to put an empty line between `ze_api.h` and `level_zero.hpp` includes:
```c++
// Native API header
#include <level_zero/ze_api.h>

// SYCL API headers
#include <CL/sycl.hpp>
#include <CL/sycl/backend/level_zero.hpp>
```